### PR TITLE
Feature/text to speech

### PR DESCRIPTION
**Pull request by tanluuuuuuu**
## Description
- Implement text to speech services using coqui-ai/TTS

## Type of change
- [x] New features (non-breaking change).
- [ ] Bug fix (non-breaking change).
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected).

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have added thorough tests if it is a core feature.
- [ ] There is a reference to the original bug report and related work.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] The feature is well documented.

### CODE REVIEW

File: apis/TTS/src/text2speech/service.py
Comment (zero-nnkn) [inline apis/TTS/src/text2speech/service.py:1]: Please update the docstring for class and functions (I think it should be "transcribe string text to audio").

**Review by nxquang-al — COMMENTED** [review]
I updated API implementation and Docker setup for deployment. It seems to work now. @tanluuuuuuu @zero-nnkn please help me review and fix them if needed.

File: apis/TTS/src/text2speech/service.py
Comment (zero-nnkn) [reply apis/TTS/src/text2speech/service.py:1]: Done

**Comment by zero-nnkn** [comment]
@nxquang-al, @tanluuuuuuu TTS will fail with empty strings. I added a preprocessing step to strip and ensure the input text of TTS is not empty.

**Comment by zero-nnkn** [comment]
Updated test for development.

**Comment by zero-nnkn** [comment]
I have checked and added test cases for this feature. Everything is OK(updated name to text-to-speech (tts) for consistency).
@tanluuuuuuu @nxquang-al, Please check for deployment.

**Review by zero-nnkn — APPROVED** [review]

**Comment by nxquang-al** [comment]
Deployed new changes successfully.

**Pull request merged**